packages/apps/graph/src/graph/objects/block.ts

```diff
@@ -8,7 +8,7 @@ import {
 import { normalizeError } from '@utils/errors';
 import { networkData } from '@utils/network';
 import { builder } from '../builder';
-import type { IGuard } from '../types/graphql-types';
+import type { Guard } from '../types/graphql-types';
```

Comment:
the only type that is not prefixed with I. Shouldnt this be IGuard, for consistency